features/php.feature

```diff
@@ -42,3 +42,19 @@ Feature: Test tasks for namespace 'php'
 		And I extend the development capistrano configuration from the fixture file server_with_faulty_domain_configuration.rb
 		When I run `cap dev php:clear_opcache`
 		Then the exit status should not be 0
+
+	Scenario: Clear APC via HTTPS
+		Given a directory named "temp"
+		Given I extend the development capistrano configuration from the fixture file server_with_ssl.rb
+		Given I set relative to current path the environment variable "SSL_CERT_FILE" to "/tmp/aruba/temp/dkdeploy.dkdeploy-php.dev.pem"
```

Comment:
We need the absolute path to the certificate file and string interpolation is possible in the cucumber step.  The RegExp match data alwasy escapes the special charachter for string interpolation